app/assets/stylesheets/darkswarm/_shop-popovers.scss

```diff
@@ -45,14 +49,15 @@
   }
 
   li {
+    line-height: 1;
     border-bottom: .1em solid $grey-700;
-    padding: 0 0.25rem;
+    padding: 0.625rem 0;
```

Comment:
should we always us .625 or 0.625. I am not sure codeclimate says something about this but we should have a standard :+1:

Reply:
Good point. We probably deactivated Code Climate for this because I remember that it preferred `.625`. I personally don't have a preference.

Reply:
ok, lets use .625 :+1: